presto-product-tests-launcher/src/main/java/io/prestosql/tests/product/launcher/env/DockerContainer.java

```diff
@@ -52,6 +52,7 @@
 import static io.airlift.concurrent.Threads.daemonThreadsNamed;
 import static io.airlift.units.DataSize.ofBytes;
 import static java.lang.String.format;
+import static java.lang.Thread.currentThread;
```

Comment:
This is typically not static imported

Reply:
`Thread.currentThread()` looks like `ImmutableList.toImmutableList()`

Reply:
I applied your comment. I see that this is common. I will leave it for another time.

Reply:
> `Thread.currentThread()` looks like `ImmutableList.toImmutableList()`

the similarity is deceptive